test/ci-operator-integration/multi-stage/configs/master/openshift-hyperkube-master.yaml

```diff
@@ -72,7 +72,7 @@ tests:
       credentials:
         - namespace: "ns"
           name: "name"
-          mountPath: "/tmp"
```

Comment:
This isn't in use yet?

/lgtm
/approve
/hold

Reply:
It is :(